src/PrestaShopBundle/Controller/Admin/Sell/Order/CartController.php

```diff
@@ -144,8 +220,53 @@ public function createAction(Request $request): JsonResponse
     }
 
     /**
-     * Changes the cart address information
+     * Exports carts
      *
+     * @param CartFilters $filters
+     *
+     * @return CsvResponse
+     */
+    public function exportAction(CartFilters $filters)
+    {
+        $filters = new CartFilters(['limit' => null] + $filters->all());
+        $cartGridFactory = $this->get('prestashop.core.grid.factory.cart');
+        $cartGrid = $cartGridFactory->getGrid($filters);
+
+        $isGuestCheckoutEnabled = $this->configuration->get('PS_GUEST_CHECKOUT_ENABLED');
+
+        $headers = [
+            'id_cart' => $this->trans('ID', 'Admin.Global'),
+            'status' => $this->trans('Order ID', 'Admin.Orderscustomers.Feature'),
```

Comment:
_Status_ or _Order ID_? It looks like the code talks about the status while the string mentions the order ID...
🤔

Reply:
Order ID - if we keep the view same as in legacy controller

Reply:
I don't know much about code but it is introduced with `'status'`, isn't it problematic?